### DISCUSSION TITLE
Remove Take A Break message?

### DISCUSSION THREAD

**Discussion by BeefMarinade** [atom/atom#22363] [Q&A] [answered]
Recently downloaded Atom and I love it.
One thing that irks me is a Take a Break message that pops up on my screen, sometimes in the middle of me typing, I've searched my packages and I haven't downloaded anything that could put that message up. Is this something in default Atom core packages?

The only community packages I added to atom are:
atom-clock-alarm
atom-focus-mode
atom-hemingway
timer (Holocron Foundation)
wordcount
I believe none of these cause a message to interupt my typing with a take a break message.

**Comment by UziTech** [comment]
I've never seen that and I'm pretty sure that is not in atom core. Is it a notification in Atom or maybe something your OS does or another program?

**Reply by BeefMarinade** [reply]
The message is attached to Atom because it stays in the Atom window as I make the screen smaller

**Comment by BeefMarinade** [comment]
![Screenshot (7974)_LI](https://user-images.githubusercontent.com/83849279/117535723-1aeba100-afc5-11eb-9db3-2448aee2a08b.jpg)
![Screenshot (7975)_LI](https://user-images.githubusercontent.com/83849279/117535728-1cb56480-afc5-11eb-8b7a-22468341c297.jpg)

**Comment by RagingTiger** [comment, marked as answer]
Apparently It's in package `atom-clock-alarm`:

Ref: https://discuss.atom.io/t/turn-off-take-a-break/71004/10

**Reply by BeefMarinade** [reply]
Thanks. Removing atom-clock-alarm fixed the problem.
